traffic_portal/app/src/common/modules/table/servers/table.servers.tpl.html

```diff
@@ -23,31 +23,47 @@
         </ol>
         <div class="pull-right">
-            <a class="btn btn-primary" name="createServersButton" title="Create Server" href="#!/servers/new"><i class="fa fa-plus"></i></a>
-            <button class="btn btn-default" title="Refresh" ng-click="refresh()"><i class="fa fa-refresh"></i></button>
-            <div id="toggleColumns" class="btn-group" role="group" title="Select Table Columns" uib-dropdown is-open="columnSettings.isopen">
-                <button type="button" class="btn btn-default dropdown-toggle" uib-dropdown-toggle aria-haspopup="true" aria-expanded="false">
-                    <i class="fa fa-columns"></i>&nbsp;
-                    <span class="caret"></span>
-                </button>
-                <menu ng-click="$event.stopPropagation()" class="column-settings dropdown-menu-right dropdown-menu" uib-dropdown-menu>
-                    <li role="menuitem" ng-repeat="c in gridOptions.columnApi.getAllColumns() | orderBy:'colDef.headerName'">
-                        <div class="checkbox">
-                            <label><input type="checkbox" ng-checked="c.isVisible()" ng-click="toggleVisibility(c.colId)">{{::c.colDef.headerName}}</label>
-                        </div>
-                    </li>
-                </menu>
-            </div>
-            <div class="btn-group" role="group" uib-dropdown is-open="more.isopen">
-                <button type="button" class="btn btn-default dropdown-toggle" uib-dropdown-toggle aria-haspopup="true" aria-expanded="false">
-                    More&nbsp;
-                    <span class="caret"></span>
-                </button>
-                <ul class="dropdown-menu-right dropdown-menu" uib-dropdown-menu>
-                    <li role="menuitem"><a ng-click="confirmCDNQueueServerUpdates(null)">Queue CDN Server Updates</a></li>
-                    <li role="menuitem"><a ng-click="confirmCDNClearServerUpdates(null)">Clear CDN Server Updates</a></li>
-                    <li role="menuitem"><button class="menu-item-button" type="button" ng-click="exportCSV()">Export CSV</button></li>
-                </ul>
+            <div class="form-inline">
+                <div class="input-group text-input">
+                    <span class="input-group-btn">
+                        <label for="quickSearch" class="btn btn-default"><i class="fa fa-search"></i></label>
+                    </span>
+                    <input id="quickSearch" type="text" class="form-control" placeholder="Quick search..." ng-model="quickSearch" ng-change="onQuickSearchChanged()"/>
+                </div>
+                <div class="input-group text-input">
+                    <span class="input-group-btn">
+                        <label for="pageSize" class="btn btn-default">Page Size:</label>
+                    </span>
+                    <input id="pageSize" type="number" class="form-control" placeholder="100" ng-model="pageSize" ng-change="onPageSizeChanged()"/>
```

Comment:
Is there a maximum we want to consider? Or a "step"? So right now if I hit the up arrow it adds one more entry per page, but I think previously it was restricted to multiples of 25.

I don't really feel strongly about having either of those, just wanted to see if you had any thoughts about them.

Also, the same `name` attribute stuff as above.

Reply:
yeah, step is probably a good idea. will add.